nemo/collections/tts/modules/submodules.py

```diff
@@ -41,11 +41,14 @@ def __init__(self, *args, **kwargs):
 
     def forward(self, input: torch.Tensor, mask_in: Tuple[int, int, int] = None):
         assert len(input.shape) == 3
-        # if a mask is input, or tensor shape changed, update mask ratio
-        if mask_in is not None or self.last_size != tuple(input.shape):
-            # borisf: disabled update for inference
-            if self.training:
-                self.last_size = tuple(input.shape)
+        # borisf: disabled cache for export
+        use_cache = not (torch.jit.is_tracing() or torch.onnx.is_in_onnx_export())
+        cache_hit = use_cache and mask_in is None and self.last_size == tuple(input.shape)
+        if cache_hit:
+            mask_ratio = self.mask_ratio
+            update_mask = self.update_mask
+            # if a mask is input, or tensor shape changed, update mask ratio
+        else:
```

Comment:
I would suggest to add a unit test to validate all possible formal logic just in case it would not crash other models.

Reply:
Partialconv is only using by RADTTS so I don't think it will create any issue. but yeah, for the future we should add unit test.